src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -106,12 +106,10 @@
   {:render
    (fn [{:keys [props state this]}]
      (cond
-       (and (:loaded-config @state)
-         (or (:workspace-id props) (:workspaces-list @state)))
-       (create-import-form state props this (:loaded-config @state) true
-         [{:label "Configuration Namespace" :key "namespace"}
-          {:label "Configuration Name" :key "name"}])
-
+       (and (:loaded-config @state) (or (:workspace-id props) (:workspaces-list @state)))
```

Comment:
If you mean that all you did is move these onto one line, that's totally fine.
I personally find it easier to read this way, but it's definitely subjective, and not a big deal:
```clojure
(and
  (statement1)
  (or (statement2) (statement3)))
```

For me, this makes it easier to see that what conditions are being compared. That said, I don't think we have a hard convention on this.